src/reviewList.jsx

```diff
@@ -0,0 +1,32 @@
+import { useState } from "react"
+function ReviewList(review_id){
+const [reviews, setReviews] = useState([])
```

Comment:
You will also want a loading state that will only render data if the loading state is false - you would set this inside the use effect to be false once the data has come through